### PR TITLE
Message tweaks

### PR DESCRIPTION
**Pull request by bendansby**
Especially mobile styles for the banner size.

Screen Shot 2021-10-03 at 11 34 52 AM
Screen Shot 2021-10-03 at 11 35 03 AM
Napkin 2 10-03-21, 12 17 26 PM

### CODE REVIEW

File: src/Nri/Ui/Message/V3.elm [hunk 1]
```diff
@@ -775,33 +800,6 @@ getRoleAttribute role =
 
 
 
--- Style overrides
```
Comment (bendansby) [inline src/Nri/Ui/Message/V3.elm:778]: I think this was only added for the login page so I will mop up the spill.

**Review by tesk9 — COMMENTED** [review]
Should the "large" banner dismiss x also have less right margin when the viewport is small?

Screen Shot 2021-10-04 at 10 02 19 AM

---

This looks good to me! I think I want to do a noredink-ui release before this merges, and right after it merges, so that some of the other changes can go out without needing to worry about Message style regressions (since this PR kindly removes the styleOverrides business 🙏 ).

**Comment by bendansby** [comment]
> Should the "large" banner dismiss x also have less right margin when the viewport is small?

Yes, because I positioned it absolutely so the text would have more room so I scooted it into the padding area to eliminate text overlap.

**Comment by bendansby** [comment]
>I think I want to do a noredink-ui release before this merges
👍

**Comment by tesk9** [comment]
Hmm, it looks to me like the "large" Message's x _isn't_ scooted and the "banner" Message's x _is_ scooted?

At any rate, feel free to merge when you're ready! (I released a noredink-ui version a few minutes ago)

**Comment by bendansby** [comment]
Good catch, unified the scoots.

**Pull request merged**